src/workers/postAddedSlackChannelSend.ts

```diff
@@ -63,8 +85,16 @@ export const postAddedSlackChannelSendWorker: TypedWorker<'api.v1.post-visible'>
               });
 
               const postLinkPlain = `${process.env.COMMENTS_PREFIX}/posts/${post.id}`;
+              const postLinkUrl = new URL(postLinkPlain);
+
+              if (adminJoinToken) {
+                postLinkUrl.searchParams.set('jt', adminJoinToken);
+                postLinkUrl.searchParams.set('source', source.handle);
+                postLinkUrl.searchParams.set('type', source.type);
+              }
+
               const postLink = addNotificationUtm(
-                postLinkPlain,
+                postLinkUrl.toString(),
```

Comment:
Let's shorten the link to obscure this info and make it clean?

Reply:
Was thinking about it, it's already obscured in the message markdown itself?

<img width="707" alt="image" src="https://github.com/user-attachments/assets/ba2c74c1-a6a4-46f6-9c23-f95fe8a21f6b">

if we don't have to add another request to shortening service maybe better, wdyt?